### DISCUSSION TITLE
[ENHANCEMENT] : Change the background

### DISCUSSION THREAD

**Discussion by PritamSarbajna** [PritamSarbajna/tourism-website#12] [Ideas]
Hey contributors change the website's main background but make sure it's tourism related. Another beautiful mountain would be great as a background.

**Comment by Iamslok** [comment]
I can help you with that, can you assign me?

**Comment by PritamSarbajna** [comment]
yeah sure @Iamslok go for it !

**Comment by Iamslok** [comment]
![stubai-glacier-snowy-mountains-wallpaper-hd-6164rdbsni545fmr](https://user-images.githubusercontent.com/90669886/184532574-adb44c24-d512-49a4-8b55-9b83a83634ef.png)
@PritamSarbajna Do you like this image?

**Comment by Iamslok** [comment]
![image](https://user-images.githubusercontent.com/90669886/184533078-ed8b6a38-a606-41c3-b5b9-9835e2757966.png)
@PritamSarbajna it will look like this!

**Comment by PritamSarbajna** [comment]
@Iamslok the 1st one is quite good but why it is faded in the second one ?

**Comment by PritamSarbajna** [comment]
If you are thinking that it'll not be looking good with this color then change the whole website's color no worries. The only goal is to make it pretty.

**Comment by Iamslok** [comment]
@PritamSarbajna The reason it's faded because of to highlight the text on top of it

**Comment by PritamSarbajna** [comment]
I thought so but with faded it's not looking good. Hey wanna make this website dark ?

**Comment by PritamSarbajna** [comment]
Yeah @Iamslok I got it man

**Comment by Iamslok** [comment]
![image](https://user-images.githubusercontent.com/90669886/184544925-0a52ea63-554a-4b1e-a0cd-26cd37c7cf3b.png)

**Comment by PritamSarbajna** [comment]
Yeah @Iamslok the text are not highlighted enough I got it.

**Comment by PritamSarbajna** [comment]
I have an idea why don't we make this website dark and the background also the background image will be dark. Not black though it's more like #333 or #222 dark. Please let me know how about that ?

**Comment by Iamslok** [comment]
@PritamSarbajna I think it's a very good idea, nowadays most people are preferring dark.

**Comment by PritamSarbajna** [comment]
I'm closing this issue and creating a new issue about dark theme do you want to work on that ?

**Comment by Iamslok** [comment]
Yeah @PritamSarbajna I would love to contribute

**Comment by PritamSarbajna** [comment]
@Iamslok here #11

**Comment by Iamslok** [comment]
@PritamSarbajna can you please assign me for #11

**Comment by PritamSarbajna** [comment]
yeah sure just make a comment there @Iamslok
